boards/sifive/hifive1/Kconfig.defconfig

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_HIFIVE1
+
```

Comment:
This is not correct, see https://github.com/nordicjm/zephyr/commits/hwmv2-riscv-1-sifive/ for how I ported it, this configuration in hwmv1 is for the revb board, not the normal board, and the config from the normal board is missing

Reply:
Thanks for the link. Should be fixed now